packages.config

```diff
@@ -14,4 +14,6 @@
   <package id="Microsoft.DiaSymReader" version="1.0.8" />
   <package id="FsSrGen" version="3.0.0" targetFramework="net46" />
   <package id="FSharp.SRGen.Build.Tasks" version="3.0.0" targetFramework="net46" />
+  <package id="NupkgMerge" version="1.0.0.1" />
+  <package id="Microsoft.FSharp.Core.netcore" version="1.0.0-alpha-160627" />
```

Comment:
package is `1.0.0-alpha-160629`, it's not in fsharp-daily feed, but is on nuget.org